pkg/state/rewards_calculator_test.go

```diff
@@ -212,7 +228,7 @@ func TestFeatures19And20And21RewardCalculation(t *testing.T) {
 		{4000, 4000, 1_9999_9999, makeTestNetRewards(t, gen, 1_9999_9999)},
 		{4000, 4000, 2_0000_0000, makeTestNetRewards(t, gen, 2_0000_0000)},
 		{4000, 4000, 4_2222_2222, makeTestNetRewards(t, gen, 3_1111_1111, 1_1111_1111)},
-		{5000, 5000, 5_0000_0000, makeTestNetRewards(t, gen, 3_5000_0000, 1_5000_0000)},
+		{4000, 4000, 5_0000_0000, makeTestNetRewards(t, gen, 3_5000_0000, 1_5000_0000)},
```

Comment:
Is it intended change? The behavior is still the same

Reply:
I think it was copy-paste error made long time ago.

Reply:
Yes, for the test behaviour is no differ on this height.

Reply:
Okay